CONTRIBUTING.md

```diff
@@ -56,33 +56,7 @@ Terraform has similar standards, and you can run `terraform fmt` to rewrite Terr
 
 ## Commit Message Format
 
-We follow a rough convention for commit messages that is designed to answer two
-questions: what changed and why. The subject line should feature the what and
```

Comment:
Chris covers this [here][1].

[1]: https://chris.beams.io/posts/git-commit/#why-not-how